src/MusicalScore/Graphical/VexFlow/VexFlowSlur.ts

```diff
@@ -66,7 +66,7 @@ export class VexFlowSlur {
     public createVexFlowCurve(): void {
             this.vfCurve = new Vex.Flow.Curve( this.vfStartNote,
                                                this.vfEndNote,
-                                               this.curve_Options()
+                                               undefined//this.curve_Options()
```

Comment:
why were the options removed? I assume this is WIP / debugging?

Reply:
This is already removed in `develop`, I think that the change comes from the rebase.